.github/workflows/auto-publish.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - run: python build.py --includes-only
```

Comment:
Running with `--include-only` won't run bikeshed, so the HTML will not be updated. Does running without this option not work?

Is there a tracking issue to get CI working for this repo?

Reply:
Updating HTML itself is being done by spec-prod internally running bikeshed.